android_jni/avifandroidjni/build.gradle

```diff
@@ -24,6 +24,7 @@ android {
     }
     externalNativeBuild {
         cmake {
+            version '3.23.3'
```

Comment:
In android_jni/avifandroidjni/src/main/jni/CMakeLists.txt, we have:

```
cmake_minimum_required(VERSION 3.7)
```

Should we change that to the actual minimum required version?

Reply:
libavif's top-level CMakeLists.txt has the following:

```
cmake_minimum_required(VERSION 3.13)
```

Reply:
Done.